iamctl/pkg/applications/export.go

```diff
@@ -96,6 +101,9 @@ func exportApp(appId string, outputDirPath string, format string, excludeSecrets
 		return fmt.Errorf("error while processing exported data: %s", err)
 	}
 
+	if excludeSecrets {
```

Comment:
Shall we mask this before processing exported content? Test out the scenario where we have added a keyword for an application's oauth secret and is exporting for the second time with exclude secrets set to true.

Reply:
changed masking before processing exported content. Also removed masked secrets before importing.